cypress-tests/cypress/e2e/PayoutUtils/Nomupay.js

```diff
@@ -0,0 +1,199 @@
+const billing = {
+    address: {
+        line1: "1467",
+        line2: "Harrison Street",
+        line3: "Harrison Street",
+        city: "Munich",
+        state: "CA",
+        zip: "80331",
+        country: "DE",
+        first_name: "John",
+        last_name: "Doe"
+    },
+    phone: {
+        number: "9123456789",
+        country_code: "+91"
+    },
+    email: `payout_customer${Date.now()}@example.com`
+};
+
+const card_data = {
+  card_number: "4111111111111111",
+  expiry_month: "3",
```

Comment:
is there a specific difference between the data used here? it is `3` here and in L29, it is `03`.

if yes, then we can keep both the objects, if not, one of them can be removed